<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finalPriceDisplay</class>
 <widget class="QDialog" name="finalPriceDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 236, 227);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 16, 84);</string>
   </property>
   <property name="text">
    <string>Service Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serviceFormAddBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(0, 170, 0);
    color: white;
    border: 1px solid green;
    border-radius: 10px;
}

/* Hover effect */
QPushButton:hover {
    background-color: rgb(0, 190, 0);
}

/* Pressed effect */
QPushButton:pressed {
    background-color: rgb(0, 140, 0);
}
</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QFrame" name="accountFormFrame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#accountFormFrame {
        border: 2px solid #1A1054;
        border-radius: 12px;
    
    }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>181</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="fnameField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mnameField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Middle Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnameField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sexField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Sex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="birthdayField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roleField_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="placeholderText">
         <string>Role</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="datehiredfield_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 16, 84);</string>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="serviceNameField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
     
	
	background-color: rgb(255, 255, 255);
    }</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="basePriceField">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
background-color: rgb(255, 255, 255);
    }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Service Name*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Base Price*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="vatCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Vat Applicable</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="finalPriceDisplay_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
background-color: rgb(255, 255, 255);
    }</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="text">
     <string>Final Price</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vatRateField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 16, 84);</string>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>12.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
